tests/PHPUnit/Domain/Exception/PreconditionExceptionUnitTest.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace PhpTuf\ComposerStager\Tests\PHPUnit\Domain\Exception;
+
+use PhpTuf\ComposerStager\Domain\Exception\PreconditionException;
+use PhpTuf\ComposerStager\Domain\Service\Precondition\PreconditionInterface;
+use PhpTuf\ComposerStager\Tests\PHPUnit\TestCase;
+use Prophecy\Prophecy\ObjectProphecy;
+
+class PreconditionExceptionUnitTest extends TestCase
+{
+    /**
+     * @covers \PhpTuf\ComposerStager\Domain\Exception\PreconditionException
+     */
+    public function test(): void
+    {
+        $precondition = $this->prophesize(PreconditionInterface::class);
+        $precondition = $precondition->reveal();
+        assert($precondition instanceof PreconditionInterface || $precondition instanceof ObjectProphecy);
```

Comment:
Well, as I recall, PhpStorm complained about using an `@var` annotation instead of that `assert()` and offered to automatically convert it. But when I went back to observe the behavior again, I couldn't reproduce. So I've reverted it since it's confusing and I prefer it the other way, too. Thanks.